onnxscript/function_libs/torch_lib/graph_building_test.py

```diff
@@ -139,5 +141,44 @@ def test_add_initializer_allows_adding_the_same_tensor_twice_using_same_name(sel
         graph.add_initializer("x", x_tensor)
 
 
+class TestLargeModelSave(unittest.TestCase):
+    def test_save_initializer_to_files_for_large_model(self):
+        class MLP(torch.nn.Module):
+            def __init__(self, input_size, hidden_size, output_size):
+                super().__init__()
+                self.fc1 = torch.nn.Linear(input_size, hidden_size)
+                self.fc2 = torch.nn.Linear(hidden_size, output_size)
+                self.relu = torch.nn.ReLU()
+
+            def forward(self, x):
+                out = self.fc1(x)
+                out = self.relu(out)
+                out = self.fc2(out)
+                return out
+
+        # # of model parameters:
+        #  input_size x hidden_size + hidden_size +
+        #  hidden_size x output_size + output_size
+        # Thus, we have
+        #  (4 x 50000000 + 50000000 = 250000000) +
+        #  (50000000 x 10 + 10 = 500000010)
+        #  = 750000010 parameters
+        #  = 750000010 * 4 bytes
+        #  = 3 GB
+        batch_size, input_size, hidden_size, output_size = 1, 4, 50000000, 10
```

Comment:
Would this consume a lot of memory? We don't have much in the CI machines. Maybe skip this in CI too?

Reply:
I will change the size of this test to `small`. Although it won't test the actual functionality, it can still test the code to fetch env variable.